core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -69,6 +69,11 @@ package object config {
     .bytesConf(ByteUnit.KiB)
     .createWithDefaultString("100k")
 
+  private[spark] val EVENT_LOG_STAGE_EXECUTOR_METRICS =
+    ConfigBuilder("spark.eventLog.logStageExecutorMetrics.enabled")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
should this be "false" for now until we could test this out more, just to be on the safe side?

Reply:
That would be safer. I'll change to false, and we can change change to true after people have had a chance to test it out.